ExtensionExamples/CustomQuestionExtensions/AdvancedCustomQuestion.cs

```diff
@@ -93,13 +93,13 @@ public override void WriteHtml(string controlPrefix, CustomQuestionProperties pr
             if (!string.IsNullOrEmpty(props.GetAttributeString(_maxLengthGuid)))
             {
                 writer.Write(" maxlength=\"");
-                writer.Write(WebUtility.HtmlEncode(props.GetAttributeString(_maxLengthGuid)));
+                writer.Write(Intelledox.Common.HtmlParsing.Sanitize(props.GetAttributeString(_maxLengthGuid)));
```

Comment:
Revert. HtmlParsing.Sanitize will return raw html which we don't want in an attribute.
Applies to the other changes as well.